include/dbstructure.php

```diff
@@ -398,6 +398,12 @@ function update_structure($verbose, $action, $tables=null, $definition=null) {
 				$sql3 .= ";";
 			}
 
+			$field_list = '';
```

Comment:
Should now be addressed accordingly https://github.com/friendica/friendica/pull/3442/commits/354531bfa6a4dab76c87b08689141056d6f439cf